api.rst

```diff
@@ -1452,6 +1452,68 @@ You can use a tool like `Swagger UI <https://swagger.io/tools/swagger-ui/>`_ to
 
   The OpenAPI information can go out of date as the schema changes under a running server. To learn how to refresh the cache see :ref:`schema_reloading`.
 
+.. _options_requests:
+
+OPTIONS
+=======
+
+You can verify which HTTP methods are allowed on endpoints for tables and views using an OPTIONS request. These methods are allowed depending on what operations *can* be done on the table or view, not on the database permissions assigned to them.
+
+CORS
+----
+
+PostgREST sets highly permissive cross origin resource sharing, that is why it accepts Ajax requests from any domain.
```

Comment:
Now that I gave it a second look. I think this should go last on the OPTIONS section - otherwise it breaks the flow of the text.